src/sdc11073/xml_types/xml_structure.py

```diff
@@ -1330,11 +1420,8 @@ def __str__(self) -> str:
 class NodeTextListProperty(_ElementListProperty):
     """The handled data is a list of words (string without whitespace). The xml text is the joined list of words."""
 
-    def __init__(self, sub_element_name: etree.QName | None,
-                 value_class: Any,
-                 is_optional: bool = False):
-        super().__init__(sub_element_name, ListConverter(ClassCheckConverter(value_class)),
-                         is_optional=is_optional)
+    def __init__(self, sub_element_name: etree.QName | None, value_class: Any, is_optional: bool = False):
+        super().__init__(sub_element_name, ListConverter(ClassCheckConverter(value_class)), is_optional=is_optional)
 
     def get_py_value_from_node(self, instance: Any, node: xml_utils.LxmlElement) -> list[Any] | None:  # noqa: ARG002
```

Comment:
typing for return value should be:

list[str]

Reply:
its `list[str] | None`, done